templates/hooks/sdk_delete_pre_build_request.go.tpl

```diff
@@ -0,0 +1,4 @@
+	if isDeleting(r) {
+		return requeueWaitWhileDeleting
```

Comment:
Did you need to `git add` a `conditions.go` file? :) I don't see where `requeueWaitWhileDeleting` is defined?

Reply:
`requeueWaitWhileDeleting` is defined in `hooks.go`

Reply:
doh! sorry, not sure how I missed that :)